src/components/tracker.js

```diff
@@ -1,28 +1,55 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { useNavigate } from 'react-router-dom';
 import { fetchTacker, setRegions } from '../Redux/tracker/tracker';
 import WorldMap from '../assets/worldmap.png';
 
-const Tacker = () => {
+const Tracker = () => {
   const dispatch = useDispatch();
+  const [searchItem, setSearchItem] = useState('');
   const trackerStore = useSelector((store) => Object.values(store.trackerReducer.tracker));
   useEffect(() => {
     if (!trackerStore.length) {
       fetchTacker(dispatch);
     }
   }, []);
   const navigate = useNavigate();
+  let alternate = true;
+  let nextLightColor = 0;
+  const Colors = (index) => {
+    if (index === nextLightColor && alternate) {
+      alternate = false;
+      nextLightColor = index + 3;
+      return 'isBlue grid-item-navlink';
+    }
+    if (index === nextLightColor && alternate === false) {
+      alternate = true;
+      nextLightColor = index + 1;
+      return 'isBlue grid-item-navlink';
+    }
+
+    return 'grid-item-navlink dark-background';
+  };
   return (
     <div>
       <div className="d-flex flex-column justify-content-center align-items-start img-class">
         <h1 className="header">Covid Tracker</h1>
         <img src={WorldMap} alt="logo" className="logo col-4 opacity-25" data-testid="img" style={{ width: '67%' }} />
       </div>
+      <input type="text" className="form-control delete-shadow" placeholder="Search by country name" onChange={(e) => {
+        setSearchItem(e.target.value);
+      }} />
```

Comment:
- [ ] Please use filter instead of search, it will be much faster to select instead of typing.

![mmmm](https://user-images.githubusercontent.com/61562626/142909904-42e32231-d8d0-4544-8585-e5e139912a55.jpeg)